python-avd/pyavd/_eos_designs/structured_config/core_interfaces_and_l3_edge/utils.py

```diff
@@ -302,18 +302,11 @@ def _get_ethernet_cfg(self: AvdStructuredConfigCoreInterfacesAndL3Edge, p2p_link
         """
         ethernet_cfg = {"speed": p2p_link.get("speed")}
 
-        if get(p2p_link, "ptp.enabled") is not True:
-            return ethernet_cfg
+        if get(p2p_link, "port_channel") is None:
+            ptp_config = self._get_interface_ptp_config(p2p_link)
 
-        ptp_config = {}
-
-        # Apply PTP profile config
-        if self.shared_utils.ptp_enabled:
-            ptp_config.update(self.shared_utils.ptp_profile)
-
-        ptp_config["enable"] = True
```

Comment:
I think we should enable PTP only when ptp_enabled on the node. Is it better to move this line in `if self.shared_utils.ptp_enabled` condition?